test/Serilog.Settings.Configuration.Tests/ConfigurationSettingsTests.cs

```diff
@@ -556,5 +558,42 @@ public void WriteToSubLoggerWithLevelSwitchIsSupported()
 
             Assert.Equal(1, DummyRollingFileSink.Emitted.Count);
         }
+
+        [Trait("Bugfix", "#103")]
+        [Fact]
+        public void MultipleArgumentValuesThrowsInvalidOperationException()
+        {
+            var jsonDiscreteValue = @"{
+                ""Serilog"": {            
+                    ""Using"": [""TestDummies""],
+                    ""WriteTo"": [{
+                        ""Name"": ""DummyRollingFile"",
+                        ""Args"": {""pathFormat"" : ""C:\\""}
+                    }]        
+                }
+            }";
+
+            var jsonComplexValue = @"{
+                ""Serilog"": {            
+                    ""Using"": [""TestDummies""],
+                    ""WriteTo"": [{
+                        ""Name"": ""DummyRollingFile"",
+                        ""Args"": {""pathFormat"" : { ""foo"" : ""bar"" } }
+                    }]        
+                }
+            }";
+
+            // These will combine into a ConfigurationSection object that has both
+            // Value == "C:\" and GetChildren() == List<string>. No configuration
+            // extension matching this exists (in theory an "object" argument could
+            // accept either value). ConfigurationReader should throw as soon as
+            // the multiple values are recognized; it will never attempt to locate
+            // a matching argument.
+
+            var ex = Assert.Throws<InvalidOperationException>(() => ConfigFromJson(jsonDiscreteValue, jsonComplexValue));
```

Comment:
can you split the line please?

Reply:
Doesn't match other `Assert.Throws` in the tests or many other lambdas throughout the project, but it doesn't bother me either way, I'll update.